ci_framework/roles/artifacts/tasks/cluster_info.yml

```diff
@@ -89,3 +89,52 @@
         chdir: "{{ cifmw_artifacts_basedir }}/logs/edpm"
       changed_when: true
       ignore_errors: true
+
+    - name: Create a file to store the operator images
+      changed_when: true
+      ignore_errors: true
+      ansible.builtin.shell: |
+        echo "cifmw_install_yamls_vars:" > {{ cifmw_artifacts_basedir }}/artifacts/operator_images.yaml
+
+    - name: Get operator images from "{{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }}" namespace
+      changed_when: true
+      ignore_errors: true
+      environment:
+        KUBECONFIG: "{{ cifmw_install_yamls_environment['KUBECONFIG'] }}"
+        PATH: "{{ cifmw_path }}"
+      ansible.builtin.shell: |
```

Comment:
using the new `ci_script` plugin may be good for that - we'd get the generated script and its logs, like we get for `ci_make`.
You can see examples of `ci_script` calls here:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/tests/integration/targets/script/tasks/main.yml

Reply:
updated